.travis.yml

```diff
@@ -124,3 +124,12 @@ after_success:
           pip install git+https://github.com/Anaconda-Server/anaconda-client
           anaconda -t ${SCIPY_WHEELS_NIGHTLY} upload --force -u ${ANACONDA_ORG} ${TRAVIS_BUILD_DIR}/wheelhouse/*.whl;
       fi
+    # for merges (push events) we use the staging area instead;
+    # SCIPY_STAGING_UPLOAD_TOKEN is a secret token used in Travis
+    # CI config, originally generated at anaconda.org for
+    # multibuild-wheels-staging
+    - if [ "$TRAVIS_EVENT_TYPE" == "push" ]; then
```

Comment:
Is this too broad? Do we want to restrict to non-master branch, since for releases the staging always happens from a named feature branch.

OTOH, may be useful for debugging off master when needed.

Reply:
Do we have an easy way to clean up uploads that are only temporarily needed for debugging?

Reply:
Anyone can go to https://anaconda.org/scipy-wheels-nightly/scipy/files, click on the "Version" and show only the files for a version. Owners also see a "Remove" button, they can remove all the files for that version.